src/main/java/io/appform/dropwizard/actors/actor/QueueType.java

```diff
@@ -0,0 +1,6 @@
+package io.appform.dropwizard.actors.actor;
+
+public enum QueueType {
+    CLASSIC,
```

Comment:
Agreed, but for replicated queues, still quorum queues are preferred
https://groups.google.com/g/rabbitmq-users/c/f3Lut7cnE4Q/m/ATUBBLiJAAAJ

We have not tested classic v2 queues because of this, hence don't want to introduce this queue type.

Reply:
Ack. Let's add it separately in another MR

Reply:
I was able to test the classic v2 queues locally hence added it's support in this PR.